package.json

```diff
@@ -212,7 +209,6 @@
     "jest": "^24.1.0",
     "jest-cli": "^24.1.0",
     "moment": "^2.29.4",
-    "moment-timezone": "^0.5.41",
```

Comment:
`moment-timezone` enriches `moment` and needs to stay.

Reply:
@AMoo-Miki dependency added back!

Reply:
I'd keep the `^` that was there before